README.md

```diff
@@ -27,6 +27,60 @@ Want to watch a screencast 🎥 about it? Check out https://symfonycasts.com/fou
 
 **[Read the Documentation](https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html)**
 
+## How to contribute
+
+The test suite of this library needs one or more database, and static analysis needs to be ran on the smaller PHP version
+supported (currently PHP 7.2), then it comes with a full docker stack.
+
+### Install docker
+
+You must [install docker](https://docs.docker.com/engine/install/) and [install docker-compose](https://docs.docker.com/compose/install/)
+at first before running the tests.
+
+### Run tests
+
+The library is shipped with a `Makefile` to run tests.
+Each target will build and start the docker stack and install composer only if needed.
+
+```shell
+$ make help
+validate                       Run full tests suite
+test-full                      Run full PHPunit (MySQL + Mongo)
+test-fast                      Run PHPunit with SQLite
+test-mysql                     Run PHPunit with mysql
+test-postgresql                Run PHPunit with postgreSQL
+test-mongo                     Run PHPunit with Mongo
+fixcs                          Run PHP CS-Fixer
+sca                            Run Psalm
+docker-start                   Build and run containers
+docker-stop                    Stop containers
+docker-purge                   Purge containers
+composer                       Run composer command
+clear                          Start from a fresh install (needed if vendors have already been installed with another php version)
+
+```
+
+You can run each `phpunit-*` target with a special argument `filter`:
+```shell
+$ make test-phunit filter=FactoryTest
+```
+
+which will use phpunit's `--filter` option.
+
+### Change docker's ports
+
+You can create a `.env` file to change the ports used by docker:
+```dotenv
```

Comment:
This is a nice touch!

Curious, how does this work? I don't see anything in the Makefile that loads this.

Reply:
docker-composer autoaticaly loads .env file if it exists